.yamato/csharp-tests.yml

```diff
@@ -12,6 +12,7 @@ test_mac_editmode_{{ editor.version }}:
   variables:
```

Comment:
Is the yml extension different from yaml ? We should try to stay consistent in the repo.

Reply:
I think `yml` is the preferred extension.

Reply:
This is consistent with the other yamato config files.

Reply:
I'm considering this out of scope for this PR.